install.sh

```diff
@@ -101,13 +101,13 @@ adjust_bin() {
     # We could add --set-rpath too, but then debugedit (called by rpmbuild) barfs
     # on the result. So use LD_LIBRARY_PATH in the thunk, below.
     patchelf \
-	--set-interpreter "$prefix/libreloc/ld.so" \
+	--set-interpreter "$root/$prefix/libreloc/ld.so" \
 	"$root/$prefix/libexec/$bin"
     cat > "$root/$prefix/bin/$bin" <<EOF
 #!/bin/bash -e
-export GNUTLS_SYSTEM_PRIORITY_FILE="\${GNUTLS_SYSTEM_PRIORITY_FILE:-$prefix/libreloc/gnutls.config}"
-export LD_LIBRARY_PATH="$prefix/libreloc"
-exec -a "\$0" "$prefix/libexec/$bin" "\$@"
+export GNUTLS_SYSTEM_PRIORITY_FILE="\${GNUTLS_SYSTEM_PRIORITY_FILE:-$root/$prefix/libreloc/gnutls.config}"
+export LD_LIBRARY_PATH="$root/$prefix/libreloc"
+exec -a "\$0" "$root/$prefix/libexec/$bin" "\$@"
 EOF
```

Comment:
Again, more consistent, but less working.

Reply:
So how does https://github.com/scylladb/scylla/blob/6c4ad932969749ee132c3a13b3c79328a1c575b5/install.sh#L103-L105 work?
```
103     patchelf \
104         --set-interpreter "$prefix/libreloc/ld.so" \
105         "$root/$prefix/libexec/$bin"
```

Reply:
Is the intention that `prefix` would be related to the runtime root and the `--root` directory be used only at install time?

Reply:
Yes, a two-step installation with $root being a stand-in for the final /.